### PR TITLE
update docs links and add another

### PR DESCRIPTION
**Pull request by zarsky-broad**

### CODE REVIEW

File: src/components/Router.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ export default class Router extends Component {
         div(['Welcome to the new FireCloud interface, powered by Terra. All of your workspaces are available. ',
           link({
             ...Utils.newTabLinkProps,
-            href: 'https://broadinstitute.zendesk.com/hc/en-us/sections/360003528231-FireCloud-users-Find-out-what-s-new-in-Terra',
+            href: 'https://support.terra.bio/hc/en-us/sections/360003528231-FireCloud-users-Find-out-what-s-new-in-Terra',
```
Comment (katevoss) [inline src/components/Router.js]: This is a broken link but the issue is the original link is broken, not the redirect. I'll find the correct new url and add a suggestion here, so please hold merging until we can fix this.

File: src/libs/ajax.js [hunk 2]
```diff
@@ -207,7 +207,7 @@ const User = signal => ({
   // 4. Reply externally (ask one of the Comms team with Full Agent access) and make sure you receive an email
   createSupportRequest: async ({ name, email, currUrl, subject, type, description, attachmentToken, emailAgreed }) => {
     return fetchOk(
-      `https://broadinstitute.zendesk.com/api/v2/requests.json`,
+      `https://support.terra.bio/api/v2/requests.json`,
```
Comment (katevoss) [inline src/libs/ajax.js:210]: I just tested this and the upload url, and it does work.

File: src/pages/LandingPage.js [hunk 3]
```diff
@@ -49,16 +59,14 @@ const LandingPage = pure(() => {
...
         div(`${getAppName()} is a cloud-native platform for biomedical`),
         div(['researchers to access ', span({ style: styles.heavy }, 'data'), ', run analysis ', span({ style: styles.heavy }, 'tools'), ',']),
         div(['and', span({ style: styles.heavy }, ' collaborate'), '.'])
       ]),
-      isFirecloud() && link({
-        href: 'https://broadinstitute.zendesk.com/hc/en-us/articles/360022694271-Side-by-side-comparison-with-Terra',
-        ...Utils.newTabLinkProps,
-        style: { margin: '1rem 0', fontSize: 18, display: 'inline-flex', alignItems: 'center' }
-      }, ['Already a FireCloud user? Learn what\'s new in Terra.', icon('pop-out', { size: 18, style: { marginLeft: '0.5rem' } })]),
+      div([makeDocLink('https://support.terra.bio/hc/en-us', 'Find how-to\'s, documentation, video tutorials, and discussion forums')]),
```
Comment (katevoss) [inline src/pages/LandingPage.js:67]: Do we have video tutorials? Right now I can only find written tutorials.

File: src/components/Router.js (hunk 1 above)
Comment (katevoss) [inline src/components/Router.js]:
```suggestion
            href: 'https://support.terra.bio/hc/en-us/sections/360004482892-Terra-for-FireCloud-Users',
```

**Review by katevoss — COMMENTED** [review]
One link to fix (it was broken before), then I'll 👍, thanks!

File: src/components/Router.js (hunk 1 above)
Comment (elizzhao) [reply src/components/Router.js]: Is it supposed to be this? https://support.terra.bio/hc/en-us/articles/360022506932-Uniquely-Terra-What-s-new-and-coming-soon

File: src/pages/LandingPage.js (hunk 3 above)
Comment (elizzhao) [reply src/pages/LandingPage.js:67]: There are some gif tutorials (the free credits stuffs)--do those count as video?

File: src/components/Router.js (hunk 1 above)
Comment (katevoss) [reply src/components/Router.js]: This is what Comms recommended, I could go either way. Maybe @bradtaylor could weigh in?

Comment (elizzhao) [reply src/components/Router.js]: Brad wants the overall page rather than the specific article! (Kate's link)

**Pull request merged**